actions/ExtraActivityAction.php

```diff
@@ -0,0 +1,240 @@
+<?php
+
+use YesWiki\Core\YesWikiAction;
+use YesWiki\Lms\Course;
+use YesWiki\Lms\ExtraActivityLog;
+use YesWiki\Lms\Module;
+use YesWiki\Lms\Service\CourseManager;
+use YesWiki\Lms\Service\ExtraActivityManager;
+use YesWiki\Lms\Service\LearnerManager;
+
+class ExtraActivityAction extends YesWikiAction
+{
+    protected $arguments;
```

Comment:
le nom $arguments n'est plus adapté car YesWikiPerformable l'utilise déjà pour gérer les arguments donnés avec l'action.
ça serait plus $parameters

Reply:
actuellement ProgressDashboard et LearnerDashboard ne gèrent pas les arguments, et ne les transmettent pas. Il faudrait soit faire en sorte qu'ils les transmettent, soit gérer uniquement les paramètres.

Reply:
J'ai résolu la conversation car ExtraActivityAction est une action est dans YesWiki, maintenant, les paramètres sont passés dans `$this->arguments`. Certes, ça ne ressemble pas aux actions créées cette hiver car à l'époque `$this->arguments` n'était pas totalement fonctionnel.